weavertest/deployer.go

```diff
@@ -147,10 +138,8 @@ func newDeployer(ctx context.Context, t testing.TB, wlet *protos.WeaveletSetupIn
 	}
 
 	t.Cleanup(func() {
-		// When the unit test ends, kill all subprocs and stop all goroutines.
-		cancel()
-		if err := d.Stop(); err != nil {
-			d.logger.Error("Stop", err)
+		if err := d.cleanup(); err != nil {
+			d.logger.Error("Wait", err)
```

Comment:
Should we rename `cleanup` to `wait`?

Reply:
In this case, the functionality is of actual cleanup, since the test has completed.
Leaving as-is, unless you have a strong preference.

Reply:
Nope, sounds good.